cluster-scope/overlays/nerc-ocp-infra/kustomization.yaml

```diff
@@ -27,6 +27,7 @@ resources:
 - apiserver/cluster.yaml
 - metrics-bucket-storage
 - redhatcop.redhat.io/odf-node-patcher.yaml
+- configmaps
```

Comment:
I think I might put this into a "grafana-dashboards" directory rather than a generic "configmap" directory.

Reply:
done